fearless/Modules/NewWallet/SendFlow/ChooseRecipient/ChooseRecipientViewLayout.swift

```diff
@@ -109,12 +122,19 @@ final class ChooseRecipientViewLayout: UIView {
             make.height.equalTo(LayoutConstants.searchViewHeight)
         }
 
-        addSubview(tableView)
-        tableView.snp.makeConstraints { make in
-            make.top.equalTo(searchView.snp.bottom)
-            make.leading.trailing.bottom.equalToSuperview()
+        let contentVStackView = UIFactory.default.createVerticalStackView()
+
+        addSubview(contentVStackView)
+        contentVStackView.snp.makeConstraints { make in
+            make.top.equalTo(searchView.snp.bottom).offset(UIConstants.verticalInset)
+            make.leading.trailing.equalToSuperview().inset(UIConstants.horizontalInset)
         }
 
+        scamWarningView.isUserInteractionEnabled = true
```

Comment:
Remove .isUserInteractionEnabled and .isHidden to scamWarningView var init

Why you should set user interaction?